src/Servers/Kestrel/Core/test/AddressBinderTests.cs

```diff
@@ -65,21 +65,20 @@ public void ParseAddressDefaultsToAnyIPOnInvalidIPAddress(string host)
         [Fact]
         public void ParseAddressLocalhost()
         {
-            var options = new KestrelServerOptions();
             var listenOptions = AddressBinder.ParseAddress("http://localhost", out var https);
             Assert.IsType<LocalhostListenOptions>(listenOptions);
-            Assert.Equal(ListenType.IPEndPoint, listenOptions.Type);
+            Assert.IsType<IPEndPoint>(listenOptions.EndPoint);
             Assert.Equal(IPAddress.Loopback, listenOptions.IPEndPoint.Address);
             Assert.Equal(80, listenOptions.IPEndPoint.Port);
             Assert.False(https);
         }
 
-        [Fact]
+        [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, WindowsVersions.Win8, WindowsVersions.Win81, WindowsVersions.Win2008R2, SkipReason = "UnixDomainSocketEndPoint is not supported on older versions of Windows")]
```

Comment:
This is breaking change as well, you can no longer construct a `UnixDomainSocketEndPoint` on an OS that doesn't support it.

Reply:
This is a rare case, correct?

Reply:
Correct, it would have failed later anyways.